examples/receive-email-using-client/receive_email_using_client.md

```diff
@@ -1,6 +1,6 @@
 # Email client - Receive email
 
-The email client is used to receive (with POP3 or IMAP4) emails using the SSL or STARTTLS protocols. This sample includes receiving emails with default configurations over SSL using the default ports via POP3. Since, IMAP4 has similar syntax we could replace `POP3 client` with `IMAP client`.
+An email server supports either POP3 or IMAP as mail access protocol. The Ballerina `email` module provides support for POP3 and IMAP mail access clients to read data from an email server. Configure either an `email:PopClient` or an `email:ImapClient` and invoke `receiveMessage` client function to retrieve data from the server. 
```

Comment:
I believe the example is written using `email:PopClient`. So description should be based on pop client.

The `email:PopClient` can receive emails via POP3 protocol. To connect to the server it requires hostname, username and password as the mandatory arguments. Once connected, `receiveMessage` method is used receive email from the server. It is a blocking method with a default timeout. 

> Note: The Ballerina `email` module also provides `email:ImapClient`, which can be used in a similar fashion. The only difference is the `email:ImapClient` uses the IMAP protocol for the communication.

Reply:
Fixed with:
- [8cc402cceb100804c7c5451081ef4f0701dc64e1](https://github.com/ballerina-platform/ballerina-distribution/pull/3863/commits/8cc402cceb100804c7c5451081ef4f0701dc64e1)
- [9f3b4631737750fdf69b10c4dd38f76431414582](https://github.com/ballerina-platform/ballerina-distribution/pull/3863/commits/9f3b4631737750fdf69b10c4dd38f76431414582)